lib/BackgroundJob/CleanUp.php

```diff
@@ -29,21 +29,29 @@ class CleanUp extends TimedJob {
 	 * @var AccessTokenMapper
 	 */
 	protected $accessTokenMapper;
+
 	/**
 	 * @var AuthorizationCodeMapper
 	 */
 	protected $authorizationCodeMapper;
 
+	/**
+	 * @var RefreshTokenMapper
+	 */
+	protected $refreshTokenMapper;
+
 	/**
 	 * Cron interval in seconds
 	 */
 	protected $interval = 86400;
 
 	public function __construct(
 		AuthorizationCodeMapper $authorizationCodeMapper,
-		AccessTokenMapper $accessTokenMapper) {
+		AccessTokenMapper $accessTokenMapper,
+		RefreshTokenMapper $refreshTokenMapper) {
```

Comment:
Missing namespace include, so this is not resolved - see drone test results. 

Needs `use OCA\OAuth2\Db\RefreshTokenMapper` at the top

Reply:
Oh, you're right. I forgot this line. 😞